app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,19 @@
 <resources>
     <string name="app_name">HWOne</string>
+    <string name="settings">Settings</string>
+    <string name="log_out">Log out</string>
+    <string name="view_my_contacts">VIEW MY CONTACTS</string>
+    <string name="edit_profile">Edit profile</string>
+    <string name="first_name_sure_name">Lucile Alvarado</string>
+    <string name="career">Graphic designer</string>
+    <string name="home_address">5295 Gaylord Walks Apk. 110</string>
+    <string name="facebook_description">Facebook</string>
+    <string name="linkedin_description">LinkedIn</string>
+    <string name="instagram_description">Instagram</string>
+
+    <string name="let_get_acquainted">Let\'s get acquainted</string>
+    <string name="fill_out_the_profile">Fill out the profile and go to the application!</string>
+    <string name="email">E-mail</string>
```

Comment:
not obvious where this string res is used, I would start string res namings with the screen they are on, and mentioning the way they are used, e.g. hint, label, error, placeholder

![image](https://github.com/user-attachments/assets/fa9f63d7-c87e-4c83-a11b-aa0c227458b6)

Reply:
done